pkg/api/controller.go

```diff
@@ -1626,23 +1626,36 @@ func (c *Controller) validateStorageNamespace(storageNamespace string) error {
 }
 
 func (c *Controller) ensureStorageNamespace(ctx context.Context, storageNamespace string) error {
-	const (
-		dummyKey  = "dummy"
-		dummyData = "this is dummy data - created by lakeFS in order to check accessibility"
-	)
+	dummyKey := fmt.Sprintf("%s/dummy", c.Config.Committed.BlockStoragePrefix)
+	const dummyData = "this is dummy data - created by lakeFS in order to check accessibility"
```

Comment:
If we're making changes...
```suggestion
	const dummyData = "this is dummy data - created by lakeFS to check accessibility"
```

Reply:
breaking backward compatibility with transient storage but we can live with that 😃